graphql/src/adapters/storage/rdbms.py

```diff
@@ -0,0 +1,45 @@
+import logging
+from abc import abstractmethod
+
+from orca_shared.database import shared_db
+from sqlalchemy import text, create_engine
+from sqlalchemy.exc import ProgrammingError
+from sqlalchemy.future import Engine
+from src.use_cases.adapter_interfaces.storage import StorageMetadataInterface
+
+
+class StorageAdapterRDBMS(StorageMetadataInterface):
+    def __init__(self, user_connection_uri: str):
+        self.user_engine: Engine = create_engine(user_connection_uri, future=True)
+
+    @shared_db.retry_operational_error()
```

Comment:
Card for potential removal of shared_db: https://bugs.earthdata.nasa.gov/browse/ORCA-589